tests/std/tests/P0896R4_views_reverse/test.cpp

```diff
@@ -317,6 +317,12 @@ using move_only_view = test::range<Category, const int, test::Sized{is_random},
     IsCommon, test::CanCompare{derived_from<Category, forward_iterator_tag>},
     test::ProxyRef{!derived_from<Category, contiguous_iterator_tag>}, test::CanView::yes, test::Copyability::move_only>;
 
+void test_gh2312() { // COMPILE-ONLY
+    using X = ranges::iota_view<int, int>;
+    ranges::reverse_view<X> view;

```

Comment:
This comes close to shadowing `ranges::view`:
https://github.com/microsoft/STL/blob/ee5a216b516ef033d9153b324842369eee7653d1/stl/inc/xutility#L2701-L2702

In general, I recommend avoiding top-level Standard identifiers. This improves codebase searchability, and makes tests quickly scannable by humans (e.g. if I see `vector` or `string` in a test, I can assume I'm looking at the Standard one).

However, as this doesn't actually shadow, no change requested - I think this is sufficiently small that a rename is not worth resetting testing.